internal/lsp/server.go

```diff
@@ -639,3 +712,18 @@ func ptrIsTrue(v *bool) bool {
 	}
 	return *v
 }
+
```

Comment:
Same as in `completions.go`: we were accessing several properties of the initialization params without first checking for `nil`, now we are not.